docs/AQL/master03-syntax.adoc

```diff
@@ -2,7 +2,13 @@
 
 == Overview
 
-AQL has five clauses: `SELECT`, `FROM`, `WHERE`, and `ORDER BY`. The `SELECT` clause specifies the data elements to be returned. The `FROM` clause specifies the result source and the corresponding containment criteria. The `WHERE` clause specifies data value criteria within the result source. The `ORDER BY` clause indicates the data items used to order the returned result set.
+AQL has the following clauses:
+
+* The `SELECT` clause specifies the data elements to be returned.
+* The `FROM` clause specifies the result source and the corresponding containment criteria.
+* The `WHERE` clause specifies data value criteria within the result source.
+* The `ORDER BY` clause indicates the data items used to order the returned result set.
+* The `LIMIT` and `OFFSET` clause indicates which portion of the result set will be returned.
```

Comment:
Probably here I would just say 'the LIMIT clause', since the OFFSET part is just part of that and it is explained that way elsewhere.

Reply:
done